docs/JestObjectAPI.md

```diff
@@ -96,18 +106,12 @@ _Note: this method was previously called `autoMockOn`. When using `babel-jest`,
 
 ### `jest.createMockFromModule(moduleName)`
 
-##### renamed in Jest **26.0.0+**
-
-Also under the alias: `.genMockFromModule(moduleName)`
```

Comment:
Hm.. Isn’t it deprecated? https://github.com/facebook/jest/blob/2fa8763db2e661b520ebebeef45a22897d6da673/packages/jest-environment/src/index.ts#L139-L141

Reply:
By the way, perhaps I could add a 'TODO remove in Jest 30' comment?

Reply:
yeah 👍